cats/tests.py

```diff
@@ -0,0 +1,25 @@
+"""
+Unit tests for the app
+"""
+import datetime
+
+from cats.models import Cat
+from cats.models import Rental
+
+
+def test_filter_by_dates(self):
```

Comment:
Where are you creating data for test? https://factoryboy.readthedocs.io/en/stable/orms.html